lib/process-services-cloud/src/lib/process/process-filters/components/edit-process-filter-cloud.component.spec.ts

```diff
@@ -500,8 +500,10 @@ describe('EditProcessFilterCloudComponent', () => {
             priority: '12',
             suspendedDateType: DateCloudFilterType.RANGE
         });
-        filter.suspendedFrom = new Date(2021, 1, 1).toString();
-        filter.suspendedTo = new Date(2021, 1, 2).toString();
+        const oneYearAgoDate = moment().add(-1, 'years').format('YYYY-MM-DD');
+        const todayDate = moment().format('YYYY-MM-DD');
```

Comment:
can we use `DATE_FORMAT_CLOUD` as format, the same as we do here -> https://github.com/Alfresco/alfresco-ng2-components/pull/7787/files#diff-84cb9bf0eff2e62913006382b1833599d9e8dd857a2c956434ab041a8f150cd7R93 ?